spec/receipt_printer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'receipt_printer'
+
+describe ReceiptPrinter do
+  subject(:receipt_printer) {described_class.new(restaurant_data, totals)}
+  let(:restaurant_data) {{:shopname => :the_coffee_connection,
+                          :address => :'123_lakeside_way',
+                          :phone => "16503600708"}}
+  let(:totals) { {subtotal: {cafe_latte: "2 x $9.50", blueberry_muffin: "1 x $4.05"}, tax: "$1.17", total: "$14.72"} }
+  context "when passed a hash of totals" do
+    before do
+      allow(Time).to receive(:new).and_return(Time.new(2016))
```

Comment:
Rails/TimeZone: Do not use `Time.new` without zone. Use one of `Time.zone.local`, `Time.new.current`, `Time.new.in_time_zone`, `Time.new.utc`, `Time.new.getlocal`, `Time.new.iso8601`, `Time.new.jisx0301`, `Time.new.rfc3339`, `Time.new.to_i`, `Time.new.to_f` instead. (https://github.com/bbatsov/rails-style-guide#time, http://danilenko.org/2012/7/6/rails_timezones)